src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -1110,6 +1111,12 @@ void CouplingSchemeConfiguration::setParallelAcceleration(
       checkIfDataIsExchanged(dataID);
     }
     scheme->setAcceleration(_accelerationConfig->getAcceleration());
+
+    if (dynamic_cast<acceleration::AitkenAcceleration *>(_accelerationConfig->getAcceleration().get()) != nullptr)
+      PRECICE_WARN("You configured participant \"{}\" in a parallel-implicit coupling scheme with \"Aitken\" "
+                   "acceleration, which is known to perform bad. Consider switching to a serial-implicit coupling "
```

Comment:
```suggestion
                   "acceleration, which is known to perform bad in this setup. Consider switching to a serial-implicit coupling "
```
or `in this context`? Otherwise it sounds like Aitken is always bad.

Reply:
Then let me try to be even more explicit:
```suggestion
                   "acceleration, which is known to perform bad in parallel coupling schemes. Consider switching to a serial-implicit coupling "
```